pxr/usdValidation/usdUtilsValidators/testenv/testUsdUtilsValidators.cpp

```diff
@@ -113,11 +126,102 @@ TestPackageEncapsulationValidator()
     TF_AXIOM(errors.empty());
 }
 
+static void
+TestMissingReferenceValidator()
+{
+    UsdValidationRegistry& registry = UsdValidationRegistry::GetInstance();
+
+    // Verify the validator exists
+    const UsdValidationValidator *validator = registry.GetOrLoadValidatorByName(
+            UsdUtilsValidatorNameTokens->missingReferenceValidator);
+
+    TF_AXIOM(validator);
+
+    const UsdStageRefPtr& stage = UsdStage::CreateInMemory();
+
+    const UsdGeomXform xform = UsdGeomXform::Define(stage, SdfPath("/Xform"));
```

Comment:
Though this is fine, but you can also create a prim like:
```
prim = stage.DefinePrim("/Prim", "Xform")
```
which won't require you to include usdGeom/xform.h